openlibrary/templates/history.html

```diff
@@ -48,9 +50,17 @@ <h1><a href="$page.key">$name</a></h1>
                         <td class="history" style="vertical-align: top;">$v.ip</td>
                     <td class="comment">$:render_template("history/comment", v)</td>
                     <td class="compare" style="text-align:center;">
-                        <input type="radio" id="a$v.revision" name="a" title="Compare this version..." value="$v.revision"/>
+                        <input type="radio" id="a$v.revision" name="a" title="Compare this version..." value="$v.revision"
+                        $if v.revision == rev-1:
```

Comment:
What's rev-1 ?
If this is a variable, I can't see where it's defined. I assume this should be the item before in the loop but I'm not seeing it in the template.

If this is an operator e.g. rev - 1,  I don't think we can safely assume that the old revision is the current revision minus one. I think

Reply:
Yes,  its an operator (rev - 1). Where rev denotes the latest revision number and rev -1 the previous revision number. 
The same logic was used in inline JS

Reply:
Thanks for chatting through this with me. Yep looks like revisions are unique per page so we can rely on -1 to get the last one.

Reply:
# #
```suggestion
                        $if v.revision == rev - 1:
```
makes it clearer it's an operator.

Reply:
Done.